presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -671,9 +671,38 @@ exists as we've rolled back to the previous state.
     -----------+------+-----------+---------
     (0 rows)
 
+DELETION
```

Comment:
Looks great! 

One question: 
Would it be appropriate to consider naming this new content DELETE, and move it from the end of the page into the SQL Support heading as a subheading between ALTER TABLE and DROP TABLE?

Reply:
Thanks for the quick review! I agree that your suggestion for the placement of the section is better. I've fixed it.